extractor/filesystem/list/list.go

```diff
@@ -252,8 +253,11 @@ var (
 		winget.Name:   {winget.NewDefault},
 	}
 
-	// Secrets list extractors for credentials.
-	Secrets = initMapFromVelesPlugins([]velesPlugin{
+	SecretsExtractors = InitMap{
```

Comment:
Prefer calling SecretExtractors/SecretDetectors (without the s in Secrets)

Also add 1-1 comment starting with `// SecretExtractors ...` and `// SecretDetectors ...` that explains what these lists store.